Tests/SwiftDocCTests/Diagnostics/DiagnosticConsoleWriterDefaultFormattingTest.swift

```diff
@@ -270,7 +270,7 @@ class DiagnosticConsoleWriterDefaultFormattingTest: XCTestCase {
         let bundle = try XCTUnwrap(fs.bundles().first)
         let baseURL = bundle.baseURL
         let source = try XCTUnwrap(bundle.markupURLs.first)
-        let range = SourceLocation(line: 3, column: 18, source: source)..<SourceLocation(line: 3, column: 32, source: source)
+        let range = SourceLocation(line: 3, column: 18, source: source)..<SourceLocation(line: 3, column: 36, source: source)
```

Comment:
I changed this test because it the range was specifying a byte in the middle of the emoji, which I don't consider to be a correct diagnostic range.